.github/workflows/test.yml

```
@@ -59,21 +54,50 @@
-  # test-matrix:
-  #   name: Test with Elixir ${{ matrix.elixir }} and OTP ${{ matrix.otp }}
-  #   runs-on: ubuntu-latest
-  #   needs: test
-  #   strategy:
-  #     matrix:
-  #       elixir: ['1.13.x', '1.14.x']
-  #       otp: ['24.x', '25.x']
-  #   steps:
-  #     # Same steps as above but using matrix versions
+      - name: Restore dependencies cache
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)